src/cm/geometry.rs

```diff
@@ -23,6 +23,12 @@ impl Rect {
                 w: self.w,
                 h: self.h - h,
             },
+            Rect {
+                x: self.x,
+                y: self.y + 2 * h,
+                w: self.w,
+                h: self.h - 2 * h,
+            },
         )
```

Comment:
The layouting of shells list looks really weird to me. I'd expect it to be on the same row with regex and cmd lists.

Reply:
Fixed the layouting, now the rect is placed next to the regexes, to the right.